public/index.html

```diff
@@ -24,7 +24,8 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>GPIB v2</title>
```

Comment:
Maybe spell it out "Get Paid In Bitcoin"

Reply:
Done.